src/app/users/users-create-account/users-create-account.component.ts

```diff
@@ -134,6 +137,14 @@ export class UsersCreateAccountComponent implements OnDestroy, OnInit {
     });
   }
 
+  checkUsernameTaken(event: any) {
+    if (event.target.value.length > 0) {
```

Comment:
is this how we want to check if the username is available or now?

Please add a todo if you want to do it later.